src/constants/default-state.ts

```diff
@@ -5,6 +5,7 @@ import { LocalLogger } from '../utils/logger';
 import { LAYOUT, Mode, NAVBAR, Renderer, SIDEPANE, VEGA_LITE_START_SPEC, View } from './consts';
 
 export interface State {
+  isAuthenticated: boolean;
```

Comment:
Can we even put this in the state that is being serialized? I feel we might need to check this when the user is opening the editor. Or am I missing something?

Reply:
@domoritz Yes, we check this. Doesn't it automatically store it in localStorage?

Reply:
It does but my question is whether that is right. If we serialize this and then the user opens the editor after 10 years, we will tell them that they are logged in even if the login has timed out.

Reply:
@domoritz Whenever the editor is opened, a fetch request is made to the back-end to check if the user is logged in and accordingly the state is set.

Reply:
Okay, then let's not persists `isAuthenticated`. Can you exclude it?